h/links.py

```diff
@@ -5,7 +5,23 @@
 """
 
 
-from h._compat import urlparse
+from h._compat import urlparse, url_unquote
+
+
+def pretty_link(url):
+    """
+    Return a nicely formatted version of a URL.
+
+    This strips off 'visual noise' from the URL including common schemes
+    (HTTP, HTTPS), domain prefixes ('www.') and query strings.
+    """
+    parsed = urlparse.urlparse(url)
+    if parsed.scheme not in ['http', 'https']:
+        return url
+    netloc = parsed.netloc
+    if netloc.startswith('www.'):
+        netloc = netloc[4:]
+    return url_unquote(netloc + parsed.path)
```

Comment:
Hmm. I'm a little confused about when we should be adding code to `h.links` and when it should go in `h.presenters.AnnotationHTMLPresenter` or `DocumentHTMLPresenter`. Is there code in one of those presenter classes that could be reused here? (Or that should be changed to call this new function instead?)